packages/terra-framework-docs/src/terra-dev-site/test/collapsible-menu-view/Filepath.test.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import CollapsibleMenuView from 'terra-collapsible-menu-view';
+
+const FilepathTestDemo = () => (
```

Comment:
CollapsibleMenu shouldn't know about the Filepath component.  You should probably give the test a generic name.